server/server.js

```diff
@@ -25,7 +27,7 @@ function addFood(toAdd) {
     while(toAdd--){
         food.push({
             // make ids unique
-            id: ((new Date()).getTime() + '' + (new Date()).getMilliseconds() + '' + food.length) >>> 0,
+            id: md5((new Date()).getTime() + '' + (new Date()).getMilliseconds() + '' + food.length),
```

Comment:
May could we use uuid that are designed for that ?
https://www.npmjs.com/package/uuid